classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :student
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_animal(student)
+    @student.push(student)
+  end
```

Comment:
- [ ] Did you mean `add_student`? `add_animal` doesn't seem to be one of the methods we expect in this exercise. Please refactor the method name accordingly.

  Inside the method, we will now need to say:
  
  ```ruby
  @students.push(student)
  ```
  
  Besides the above refactoring, we also need to fulfill the requirement below: 
  
  ---
  > - Make sure that when adding a student to a classroom it also sets the classroom for the student.
  ---
  
  Please also set the classroom for the student as required in the requirement above.